helpdesk_mgmt_timesheet/models/helpdesk_ticket_team.py

```diff
@@ -8,6 +8,7 @@ class HelpdeskTicketTeam(models.Model):
     _inherit = "helpdesk.ticket.team"
 
     allow_timesheet = fields.Boolean()
+    show_timesheet = fields.Boolean(help="Show spent time of tickets in the portal")
```

Comment:
In this case, I think it's worth to be more specific in the field name.

```suggestion
    show_timesheet_portal = fields.Boolean(help="Show spent time of tickets in the portal")
```